packages/controllers/src/snaps/SnapController.test.ts

```diff
@@ -2555,6 +2704,279 @@ describe('SnapController', () => {
     });
   });
 
+  describe('isBlocked', () => {
+    it('throws an error if _checkSnapBlockList is undefined', async () => {
+      const snapController = getSnapController();
+      await expect(
+        snapController.isBlocked('npm:example', '1.0.0'),
+      ).rejects.toThrow(
+        'There is no snap block list defined for this controller.',
+      );
+    });
+
+    it('does nothing if _checkSnapBlockList is undefined', async () => {
```

Comment:
This test seems like it is labeled wrong?

Reply:
bd16c5d